### PR TITLE
Add EEA token execution logic application

### PR DESCRIPTION
**Pull request by reckeyzhang**

### CODE REVIEW

File: examples/apps/eea_token/readme [hunk 1]
```diff
@@ -0,0 +1,69 @@
+
+EEA token execution logic allows running the execution logic based on token requests,
+alse see check https://github.com/EntEthAlliance/Trusted-Token token_validation.cpp
```
Comment (manojgop) [inline examples/apps/eea_token/readme]: The link https://github.com/EntEthAlliance/Trusted-Token is broken. Is there any link where we can find more details about trusted token and about functionality implemented in this PR

Comment (danintel) [reply examples/apps/eea_token/readme]: Sanjay Bakshi may have a link.
I do not know of any.

**Comment by danintel** [comment]
The Jenkins CI build failed because you did a `merge` and the merge commit has no "DCO" line. You should use `rebase` instead of `merge`.

File: examples/apps/eea_token/readme (hunk 1 above)
Comment (reckeyzhang) [reply examples/apps/eea_token/readme]: This link is right and the github project is currently private

Comment (danintel) [reply examples/apps/eea_token/readme]: I confirmed the link is correct with Sanjay.

**Comment by reckeyzhang** [comment]
Oops, sorry about that, I redo a PR: Add EEA token execution logic
application with updated license #55


Thanks
Lei

**Pull request closed without merging**